amundsen_application/static/js/components/PreferencesPage/PreferenceGroup/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+export type PreferenceGroupProps = {
+  changePreference: any;
+  selected: boolean;
+  title: string;
+  subtitle: string;
+}
+
+export class PreferenceGroup extends React.Component<PreferenceGroupProps> {
+    public static defaultProps: Partial<PreferenceGroupProps> = {
+      selected: false,
+      title: '',
+      subtitle: '',
+    };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
```

Comment:
Can you add a `TODO: Consolidate with future common RadioButton component`. 

Context: Our new designs involve more places where users will have radio buttons and/or checkboxes, so as a result we will create some re-useable components because there is no reason for us to have to add all this markup each time we want a radio item. Ideally once we consolidate all this work over the next couple weeks, this can get replaced with a `<RadioButton>` that takes some props and children.